corehq/apps/es/migrations/0003_add_assigned_location_ids.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.16 on 2023-04-25 10:25
+
+import corehq.apps.es.migration_operations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('es', '0002_add_tombstones'),
+    ]
+
+    operations = [
+        corehq.apps.es.migration_operations.UpdateIndexMapping(
```

Comment:
@AmitPhulera is `ptop_preindex` required after this is applied? If yes and if it is not run automatically, we should have the ES migration operations request a reindex after migrations are applied, similar to how [`RequestReindex`](https://github.com/dimagi/commcare-hq/blob/8e7b380c908368492cdfb7742627ad9062052d20/corehq/preindex/django_migrations.py#L78) does.

Reply:
No. `ptop_preindex` is not required after this is applied. The relevant docs would be indexed again by the migration added by Charl in one of the previous commits.

Reply:
@AmitPhulera I should have asked one more question: what do you mean by "previous commits"? Was that on a different PR? I only see one commit on this PR.

Reply:
@millerdev I think @AmitPhulera might be referring to [this PR](https://github.com/dimagi/commcare-hq/pull/32864).